ui/raidboss/data/04-sb/ultimate/unending_coil_ultimate.ts

```diff
@@ -721,23 +754,50 @@ const triggerSet: TriggerSet<Data> = {
         },
       },
     },
+    {
+      id: 'UCU Nael Thunder Collect',
+      type: 'Ability',
+      netRegex: { source: 'Thunderwing', id: '26C7' },
+      run: (data, matches) => {
+        data.thunderDebuffs.push(matches.target);
+        if (data.me === matches.target)
+          data.thunderOnYou = true;
+      },
+    },
     {
       id: 'UCU Nael Thunderstruck',
       type: 'Ability',
       // Note: The 0A event happens before 'gains the effect' and 'starts
       // casting on' only includes one person.
-      netRegex: { source: 'Thunderwing', id: '26C7' },
-      condition: Conditions.targetIsYou(),
-      alarmText: (_data, _matches, output) => output.text!(),
+      netRegex: { source: 'Thunderwing', id: '26C7', capture: false },
+      delaySeconds: 0.5,
+      suppressSeconds: 5,
+      alarmText: (data, _matches, output) => {
+        if (data.thunderOnYou)
+          return output.thunderOnYou!();
+      },
+      infoText: (data, _matches, output) => {
+        if (!data.thunderOnYou) {
+          const thunderPlayers = data.thunderDebuffs.map((p) => data.ShortName(p)).join(', ');
+          return output.thunderOnOthers!({ players: thunderPlayers });
+        }
+      },
+      run: (data) => {
+        data.thunderDebuffs = [];
+        data.thunderOnYou = false;
+      },
       outputStrings: {
-        text: {
+        thunderOnYou: {
           en: 'Thunder on YOU',
           de: 'Blitz auf DIR',
           fr: 'Foudre sur VOUS',
           ja: '自分にサンダー',
           cn: '雷点名',
           ko: '나에게 번개',
         },
+        thunderOnOthers: {
+          en: 'Thunder on ${players}',
```

Comment:
Sorry for skepticism, but I'm just not sure why this is helpful information to know if you don't have thunder.  Mostly just don't understand what the collection is for here.

Reply:
Quickly knowing who has thunder (especially when those debuffs overlap with other Nael mechanics during quotes 3 & 4) allows you to identify on screen if those players have recognized that fact and moved away -- or, if that player also has doom, to recognize that they will be moving to cleanse (requiring the rest of the party to possibly adjust position to accommodate).  I found it helpful during prog, but ymmv, and I'm not wedded to it.

Reply:
Yeah, I am mostly worried about noise here especially with dooms going off too.  I do agree that knowing that the main tank has thunder here goes a long way and that was something that we would call out manually.  Ok, I think I've convinced myself this would be useful; I was mostly thinking about the first set.

My only other request here is, can you make two output strings for "single player thunder" and "multiple player thunder" (e.g. ${player1} ${player2}) to match other triggers? I know it looks unnecessary, but sometimes different languages might want to write this out differently, and having player name output be its own thing will make it easier for the future if we ever want to add settings for player => job.

Reply:
Just to make sure I'm understanding the suggestion: right now, there's an existing outputString for `thunderOnYou` (which will trigger if you have thunder), and this PR adds `thunderOnOthers` (which will trigger if you don't have thunder, because two other party members do).  So I think your ask is to just update `thunderOnOthers` to call out player1 and player2 separately, as opposed to being a single passed string?  I'll update it to that, but let me know if I've misunderstood the ask.

Reply:
Yeah, keep `thunderOnYou`.  Separate out `thunderOnOthers` into `thunderOnOther` (1 player) and `thunderOnOthers` (2 players) and have `thunderOnOthers` be `Thunder on ${player1}, ${player2}` rather than combined in code.

Reply:
OK, that's how I pushed it in the latest commit. 😄   Should be good to go.